Reworked riot suit, resprited helmets

TadJohnson00 opened the pull request:
Riot suit now fits palette and theme, and should be more balanced as compared to plate carrier and duravest

Also helmets now resprited

<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Riot suits are now slower, a bit more generally protective, and prettier and less confusing to look at. Also the sec and riot helmets are resprited now :3
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Needed balancing against plate carrier and duravest, resprites are just following from previous PRs
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

Hyenh commented:
These commits were already merged?

DEATHB4DEFEAT commented:
reset your branch

Jerryaqua commented:
Ah fuck why does this keep happning, how can I remove the last update, I didn't want to commit this to here

Jerryaqua commented:
githib is SO hard to use damn it

DEATHB4DEFEAT commented:
> opened 7 days ago

This hot fix is quite cold

DebugOk commented:
Issues fixed in #175

TadJohnson00 committed:
Reworked riot suit, resprited helmets

Riot suit now fits palette and theme, and should be more balanced as compared to plate carrier and duravest

Also helmets now resprited

TadJohnson00 committed:
Merge branch 'master' into riotsuit

Colin-Tel reviewed (approved):
I love beating up civilians while being physically insulated from retaliation!!
